### DISCUSSION TITLE
Code-server on your own website

### DISCUSSION THREAD

**Discussion by HalfTurk** [coder/code-server#3529] [General] [answered]
Hi there! Is it possible to run code-server on your own website? Specifically running it 24/7 and being able to use on the go with WiFi of course? I have my own site and am hosting it so would someone respond and tell me how I could achieve this?

Edit: I do NOT want to have it keep running on my computer in order to make it work if you are using the other method I saw.

Thanks in advance

**Comment by jsjoeio** [comment, marked as answer]
Certainly! The easiest solution in my opinion is to deploy it with Railway. Check out the guides in `deploy-code-server` [here](https://github.com/cdr/deploy-code-server) (shout-out to @bpmct for making this!).

**Reply by HalfTurk** [reply]
I somehow went only my email exactly one minute after you said this. Thank you. I’ll keep you posted if I have any further questions. Continue the updates! (We love code-server!)

**Reply by jsjoeio** [reply]
Ha! That's funny. No problem. Thanks for the love ♥️ We all appreciate it!
